domains/cracked.is-cool.dev.json

```diff
@@ -0,0 +1,14 @@
+{
+    "domain": "is-cool.dev",
+    "subdomain": "cracked",
+
+    "owner": {
+        "email": "cracked21@skiff.com"
+    },
+
+    "record": {
+        "MX": ["10"]
```

Comment:
Invalid MX record.

Reply:
{
    "domain": "is-cool.dev",
    "subdomain": "cracked",

    "owner": {
        "email": "cracked21@skiff.com"
    },

    "record": {
        "MX": ["inbound-smtp.skiff.com"]
    },

    "proxied": false
}